deployment/enable_maintenance_mode.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
```

Comment:
It's a good practice to have `set -e` here which makes the script execution behave as if it was started with the `-e` flag to bash, so that the script stops if any commands of the script fail. Then it would for example end in `cp: cannot create regular file '/var/www/html/maintenance.css': Permission denied` if one forgets to use `sudo` instead of proceeding and producing more output. Any reasons to not include that here?

Reply:
i'm fine with adding it :)